addons/io_hubs_addon/io/utils.py

```diff
@@ -196,7 +196,7 @@ def gather_node_property(export_settings, blender_object, target, property_name)
     blender_object = getattr(target, property_name)
 
     if blender_object:
-        if bpy.app.version < (3, 2, 0):
+        if bpy.app.version < (3, 2, 0) or bpy.app.version >= (4, 0, 0):
```

Comment:
I think >= 4.0 uses the other signature so this change is not necessary

Reply:
If I remember correctly this was the same for 3.2 and 4.x, but different in 3.5 and 3.6 and failed without that if